### DISCUSSION TITLE
MAUI Support for Linux Desktop Apps

### DISCUSSION THREAD

**Discussion by satyajit-behera** [dotnet/maui#12392] [Ideas]
Any plans to support Linux desktop apps like ubuntu? 
Is there a workaround to deploy the existing MAUI application to Ubuntu Linux?

**Comment by ScarletKuro** [comment]
Such discussion already exists https://github.com/dotnet/maui/discussions/339
There is no plan to support Linux desktop in MAUI for now

**Comment by MartyIX** [comment]
> Is there a workaround to deploy the existing MAUI application to Ubuntu Linux?

I don't think so.

**Comment by KTPenrose** [comment]
MAUI without Linux support is useless.  Its not even worth investigating.
